src/trimgalore/config.vsh.yaml

```diff
@@ -0,0 +1,314 @@
+        required: false
+        description: Output file for read 2. Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: read_2.fastq
+      - name: --trimming_report_r1
+        type: file
+        required: false
+        description: Trimming report for read 1. Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: read_1.trimming_report.txt
+      - name: --trimming_report_r2
+        type: file
+        description: Trimming report for read 1. Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        required: false
+        example: read_2.trimming_report.txt
+      - name: --trimmed_fastqc_html_1
+        type: file
+        required: false
+        description: FastQC report for trimmed (single-end) reads (or read 1 for paired-end). Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: read_1.fastqc.html
+      - name: --trimmed_fastqc_html_2
+        type: file
+        description: FastQC report for trimmed reads (read2 for paired-end). Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        required: false
+        example: read_2.fastqc.html
+      - name: --trimmed_fastqc_zip_1
+        type: file
+        required: false
+        description: FastQC results for trimmed (single-end) reads (or read 1 for paired-end). Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: read_1.fastqc.zip
+      - name: --trimmed_fastqc_zip_2
+        type: file
+        description: FastQC results for trimmed reads (read2 for paired-end). Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        required: false
+        example: read_2.fastqc.zip
+      - name: --unpaired_r1
+        type: file
+        required: false
+        description: Output file for unpired read 1. Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: unpaired_read_1.fastq
+      - name: --unpaired_r2
+        type: file
+        required: false
+        description: Output file for unpaired read 2. Only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
+        direction: output
+        example: unpaired_read_2.fastq
+
+resources:
+  - type: bash_script
+    path: script.sh
+
+test_resources:
+  - type: bash_script
+    path: test.sh
+    
+engines:
+- type: docker
+  image: quay.io/biocontainers/trim-galore:0.6.9--hdfd78af_0
```

Comment:
Please verify that this is the latest version.

Reply:
I updated the image.